clients/drcachesim/tests/offline-allasm-scattergather-basic-counts.templatex

```diff
@@ -0,0 +1,41 @@
+Correct
+Hello world!
+Basic counts tool results:
+Total counts:
+     .* total \(fetched\) instructions
```

Comment:
I assume the instruction count is way off from incorrectly separately recording all the expansion instructions, right?  Please put a FIXME i#4865.

We should have a precise number here, right, not a .*?  Ditto for the line below.

Reply:
I put the exact counts for reads and stores. Sure, I'll put those for instrs as well.

There's a FIXME already on the load/store instrs; I'll modify it to include instr counts too.

Reply:
Done.